src/controllers/authenticationController.ts

```diff
@@ -59,25 +64,23 @@ export default class AuthenticationController {
     const spidUser = errorOrUser.value;
     const sessionToken = this.tokenService.getNewToken() as SessionToken;
 
-    const errorOrResponse = await SessionStorage.set(spidUser, sessionToken);
+    const errorOption = await SessionStorage.set(
```

Comment:
for consistency with the rest of the codebase, we use the `maybe` prefix for option variables, i.e. `maybeError`